packages/hash/frontend/src/theme/shadows.ts

```diff
@@ -0,0 +1,11 @@
+import { ThemeOptions } from "@mui/material";
+
+export const shadows: ThemeOptions["shadows"] = {
+  none: "",
+  xs: "0px 1px 5px rgba(27, 33, 40, 0.07)",
+  sm: "0px 4px 11px rgba(39, 50, 86, 0.04), 0px 2.59259px 6.44213px rgba(39, 50, 86, 0.08), 0px 0.5px 1px rgba(39, 50, 86, 0.15)",
+  md: "0px 11px 30px rgba(61, 78, 133, 0.04), 0px 7.12963px 18.37px rgba(61, 78, 133, 0.05), 0px 4.23704px 8.1px rgba(61, 78, 133, 0.06), 0px 0.203704px 0.62963px rgba(61, 78, 133, 0.07)",
+  lg: "0px 20px 41px rgba(61, 78, 133, 0.07), 0px 16px 25px rgba(61, 78, 133, 0.0531481), 0px 12px 12px rgba(61, 78, 133, 0.0325), 0px 2px 3.13px rgba(61, 78, 133, 0.02)",
+  xl: "0px 51px 87px rgba(50, 65, 111, 0.07), 0px 33.0556px 50.9514px rgba(50, 65, 111, 0.0531481), 0px 19.6444px 27.7111px rgba(50, 65, 111, 0.0425185), 0px 10.2px 14.1375px rgba(50, 65, 111, 0.035), 0px 4.15556px 7.08889px rgba(50, 65, 111, 0.0274815), 0px 0.944444px 3.42361px rgba(50, 65, 111, 0.0168519)",
```

Comment:
Non-blocking, but I think it'd be more readable for these to be declared in arrays. For example:
```suggestion
xl: [
    "0px 51px 87px rgba(50, 65, 111, 0.07)",
    "0px 33.0556px 50.9514px rgba(50, 65, 111, 0.0531481)",
    "0px 19.6444px 27.7111px rgba(50, 65, 111, 0.0425185)",
    "0px 10.2px 14.1375px rgba(50, 65, 111, 0.035)",
    "0px 4.15556px 7.08889px rgba(50, 65, 111, 0.0274815)",
    "0px 0.944444px 3.42361px rgba(50, 65, 111, 0.0168519)",
  ].join(","),
```

Reply:
Noted. Added in 877d708